### ISSUE TITLE
Sync Batch Norm 

### ISSUE THREAD

**Issue by thangvubk** [ShuLiu1993/PANet#3]
Thank you for your sharing. 
By quick search on github i found that you open an issue in Sync Batch Norm repo: 
https://github.com/vacancy/Synchronized-BatchNorm-PyTorch/issues/3
Is this the main reason that sync batch norm is not supported in this PANet repo?

**Comment by thangvubk** [comment]
Thank you for your information. Hope to see your result soon.
